README.md

```diff
@@ -43,12 +69,14 @@ k8s/bin/tf destroy -force
 setup/cleanup.sh
 ```
 
+If not, please follow our step-by-step guide below.
```

Comment:
no need for "please" here, how about this :

Otherwise follow our step-by-step guide below.